core/network-peerset/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+description = "Connectivity manager based on reputation"
+homepage = "http://parity.io"
+license = "GPL-3.0"
+name = "substrate-network-peerset"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+fnv = "1.0"
+libp2p = { version = "0.4.0", default-features = false }
```

Comment:
is this still current or should we switch this to 0.5 now that you've released that?